src/column.rs

```diff
@@ -29,7 +29,7 @@ use crate::{
 use crate::compress::Compress;
 
 const START_BITS: u8 = 16;
-const MAX_REBALANCE_BATCH: u32 = 1024;
+const MAX_REBALANCE_BATCH: usize = 8192;
```

Comment:
This is now counted in entries, rather than chunks

Reply:
Depends on how full the chunks would be.  The problem with the previous chunk-based approach is that when reindexing a mostly-empty index it would produce empty batches and reindexing would stop.